src/components/Table/columnSizes.ts

```diff
@@ -25,8 +25,8 @@ import { useDebouncedCallback } from "use-debounce";
 export function useSetupColumnSizes(
   style: GridStyle,
   columns: GridColumn<any>[],
-  tableRef: MutableRefObject<HTMLElement | null>,
-  resizeTarget: MutableRefObject<HTMLElement | null> | undefined,
+  resizeRef: MutableRefObject<HTMLDivElement | null>,
+  externalResizeTarget: MutableRefObject<HTMLElement | null> | undefined,
```

Comment:
Hm, should we have just 1 `resizeRef` param here, and have line 325 be:

```
const columnSizes = useSetupColumnSizes(style, columns, resizeRef ?? resizeTarget);
```

?

AFAICT we only ever use 1 or the other in this method?

Reply:
True, will update